odoo/models.py

```diff
@@ -1078,6 +1078,7 @@ def _validate_fields(self, field_names):
                 except ValidationError, e:
                     raise
                 except Exception, e:
+                    _logger.error('Exception while validating constraint', exc_info=True)
```

Comment:
Looks fine to me :+1:
Is there any reason to prefer `logger.error(msg, exc_info=True)` to `logger.exception(msg)`, though?

Reply:
AFAIK, `exc_info=True` logs the traceback, and it's the main reason behind this.

Reply:
I thought that logs with level EXCEPTION, but the docs effectively say it's the same.